.circleci/README.md

```diff
@@ -1,22 +1,25 @@
 ## Setting up CircleCI for Scalar DL Node/Web client SDK
 
-### 1. Environment Variables
+### 1.0. Environment Variables
```

Comment:
Instead of numbering, why don't you use `###` and `####` ?

Reply:
Okay, I'll just remove the numbers.